src/Trace/Integrations/PDO.php

```diff
@@ -31,11 +31,19 @@
  */
 class PDO implements IntegrationInterface
 {
+    // database connection string dsn
     static $dsn = "";
+
+    // optional parameters
+    // - proxy_sql: bool - if connection was made using proxy_sql
+    static $options = [];
+
     /**
      * Static method to add instrumentation to the PDO requests
+     * @param string $dsn
+     * @param array $options
      */
-    public static function load($dsn="")
+    public static function load($dsn = "", $options = [])
```

Comment:
Was suggesting `Options` to have `tags` just like [RequestHandler](https://github.com/razorpay/opencensus-php/blob/master/src/Trace/RequestHandler.php#L94)'s options have `headers`. 

then [here](https://github.com/razorpay/opencensus-php/pull/17/files#diff-b20a7af510d887c9b981d5b3103820e9f456d77ed91e47b6a3cceaef89f99eefR135), you could simply add $options['tags'] to existing $attributes.